Jenkinsfile

```diff
@@ -29,225 +23,154 @@ pipeline {
+            }
+            post {
+                success {
+                    colourText("info","Stage: ${env.STAGE} successful!")
+                }
+                failure {
+                    colourText("warn","Stage: ${env.STAGE} failed!")
                 }
             }
         }
 
-        stage('Static Analysis') {
+        stage('Test'){
             agent any
+            environment{ STAGE = "Test"  }
             steps {
-                parallel (
-                        "Unit" :  {
-                            colourText("info","Running unit tests")
-                            sh "$SBT test"
-                        },
-                        "Style" : {
-                            colourText("info","Running style tests")
-                            sh """
-                                $SBT scalastyleGenerateConfig
-                                $SBT scalastyle
-                            """
-                        },
-                        "Additional" : {
-                            colourText("info","Running additional tests")
-                            sh "$SBT scapegoat"
-                        }
-                )
+                colourText("info", "Building ${env.BUILD_ID} on ${env.JENKINS_URL} from branch ${env.BRANCH_NAME}")
+                sh 'sbt coverage test coverageReport'
             }
             post {
-                always {
-                    script {
-                        env.NODE_STAGE = "Static Analysis"
-                    }
-                }
                 success {
-                    colourText("info","Generating reports for tests")
-                    //   junit '**/target/test-reports/*.xml'
-
-                    step([$class: 'CoberturaPublisher', coberturaReportFile: '**/target/scala-2.11/coverage-report/*.xml'])
-                    step([$class: 'CheckStylePublisher', pattern: 'target/scalastyle-result.xml, target/scala-2.11/scapegoat-report/scapegoat-scalastyle.xml'])
+                    junit '**/target/test-reports/*.xml'
+                    step([$class: 'CoberturaPublisher', coberturaReportFile: '**/target/*/coverage-report/cobertura.xml'])
+                    colourText("info","Stage: ${env.STAGE} successful!")
                 }
                 failure {
-                    colourText("warn","Failed on Tests.")
+                    colourText("warn","Stage: ${env.STAGE} failed!")
                 }
             }
         }
 
-
-        // bundle all libs and dependencies
-        stage ('Bundle') {
+        stage('Static Analysis') {
             agent any
-            when {
-                anyOf {
-                    branch BRANCH_DEV
-                    branch BRANCH_TEST
-                    branch BRANCH_PROD
-                }
-            }
+            environment{ STAGE = "Static Analysis" }
             steps {
-                script {
-                    env.NODE_STAGE = "Bundle"
+                parallel (
```

Comment:
This is the scripted style syntax. If you want to use declarative then see here: 
https://jenkins.io/doc/book/pipeline/syntax/#parallel

Reply:
Done